### PR TITLE
Updating the crispr parser to accommodate the new json schema

### PR DESCRIPTION
**Pull request by DSuveges**
So the CRISP parser is rewritten to generate evidence for the new schema.

Call the script:
```bash
(venv)$ python3 modules/CRISPR.py -e resources/crispr_evidence.tsv \
    -d resources/crispr_descriptions.tsv_ \
    -c resources/crispr_cell_lines.tsv   \
    -o crispr.json.gz 
```

There were quite many changes in the flow of the script:
*  There's no check with the `geneParser` all genes are pushed forward to the pipeline. 
* The output is a gzipped json.
* Log are printed to `stderr` (or optionally into a file)
* The pubmed reference is removed as that was a resource specific reference not an evidence specific.

This is the resulting structure looks like this:

```json
{
  "targetFromSourceId": "ENSG00000110092",
  "diseaseFromSourceMappedId": "EFO_0000174",
  "resourceScore": 0.797813,
  "diseaseFromSource": "Bone",
  "diseaseCellLines": [
    "CAL-72",
    "ES4",
    "ES5",
    "ES8",
    "EW-1",
    "EW-16",
    "EW-22",
    "EW-7",
    "MC-IXC",
    "MHH-ES-1",
    "SJSA-1",
    "TC-71"
  ],
  "datasourceId": "crispr",
  "datatypeId": "affected_pathway"
}
```
All the generated evidence validate against the most recent schema.

### CODE REVIEW

**Review by ireneisdoomed — COMMENTED** [review]
The evidence have been successfully validated. I have particularly enjoyed line 124 and the indicator check in line 164.

My main concern is about calculating the score within the parser. We may better speak with BE or PR them. 
Also, I would remove the legacy references to CRISPR in `settings.py` and reconsider the presence of the input files in the `resources` folder.

File: modules/CRISPR.py [hunk 1]
```diff
@@ -32,198 +54,141 @@
...
+        .drop(['pmid','gene_set_name', 'disease_name'], axis=1)
+        .rename(columns={
+            'target_id': 'targetFromSourceId', 
+            'disease_id':'diseaseFromSourceMappedId',
+            'score': 'resourceScore',
+        })
+    )
+    
+    # Replace some target ids: 
+    evidence_df.targetFromSourceId = evidence_df.targetFromSourceId.apply(lambda x: CRISPR_SYMBOL_MAPPING[x] if x in CRISPR_SYMBOL_MAPPING else x)
```
[the remark this answers is not in the document]
Comment (DSuveges) [reply modules/CRISPR.py:154]: I checked this, and some of those labels are not symbols but synonyms. I think at this point only uniport ids, and hgnc symbols are mapped to gene ids. We have to keep it for now. I don't like it either.

File: modules/CRISPR.py [hunk 2]
```diff
@@ -32,198 +54,141 @@
...
+
+    # Convert score:
+    evidence_df.resourceScore = evidence_df.resourceScore/100
```
[the remark this answers is not in the document]
Comment (DSuveges) [reply modules/CRISPR.py:157]: OK, I remove this and will let Miguel know that a new rule needs to be added to the config.

File: modules/CRISPR.py [hunk 3]
```diff
@@ -1,23 +1,45 @@
...
-__credits__   = ["Michaela Spitzer", "Asier Gonzalez"]
-__license__   = "Apache 2.0"
-__version__   = "1.3.0"
-__maintainer__= "Open Targets Data Team"
-__email__     = ["data@opentargets.org"]
-__status__    = "Production"
-
-CRISPR_DATABASE_ID='Project Score'
-CRISPR_VERSION='April 2019'
+'''
```
[the remark this answers is not in the document]
Comment (DSuveges) [reply modules/CRISPR.py:7]: You are right, I just pasted that so I know what is the expected output. :D

**Review by ireneisdoomed — APPROVED** [review]
Changes reviewed and validated.

**Pull request merged**